### DISCUSSION TITLE
EPROM  HAT

### DISCUSSION THREAD

**Discussion by zano112** [GlasgowEmbedded/glasgow#572] [General]
Hello 

Is  there  any  direct  link  to  the  EPROM  HAT ( pcb and  example )   that  is  used  on  GLASGOW   ?

**Comment by esden** [comment]
It has been a while since I worked with PROMs but here is an adapter hat that I designed: https://github.com/esden/glasgow-addons/tree/master/hardware/prom-addon it should work with the prom applet.

**Comment by zano112** [comment]
Thank  you  very  much  .
